README.md

```diff
@@ -64,6 +64,29 @@ We invest a lot of time developing, maintaining and updating our awesome open-so
 
 Let's take these great projects to the next level together.
 
+## FAQ
+
+**How do I access clue·access Projects?** – 
+*This is a manual process, where you will be invited to join clue·access on GitHub,
+thereby becoming part of the sponsors team. This means that all the projects in clue·access are
+available to you.*
```

Comment:
Good progress! Perhaps this can be rephrased to focus on the happy path before going into the details of the manual process (already answered in next question)?

Reply:
Slightly rephrased.